index.html

```diff
@@ -190,4 +190,4 @@
 		</div>
 	</article>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
I read through the linked StackOverflow article and it seems like its better to be safe than sorry in this case (I'm also pretty happy to get rid of that icon at the end of my code!).